### PR TITLE
Fix serialization Providers

### PR DESCRIPTION
**Pull request by BillHiebert**
I had defined the serialization providers incorrectly. Verified this new definition works correctly.

### CODE REVIEW

**Comment by BillHiebert** [comment]
@abpiskunov  - I am oof next week. Can you push this through

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ internal class LaunchSettingsProvider : OnceInitializedOnceDisposed, ILaunchSett
             ProjectServices = projectServices;
             FileManager = fileSystem;
             CommonProjectServices = commonProjectServices;
-            JsonSerializationProviders = new OrderPrecedenceImportCollection<Lazy<ILaunchSettingsSerializationProvider, IJsonSection>>(ImportOrderPrecedenceComparer.PreferenceOrder.PreferredComesFirst, 
+            JsonSerializationProviders = new OrderPrecedenceImportCollection<ILaunchSettingsSerializationProvider, IJsonSection>(ImportOrderPrecedenceComparer.PreferenceOrder.PreferredComesFirst, 
                                                                                                                     unconfiguredProject);;
```
Comment (abpiskunov) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs]: >; [](start = 137, length = 1)

nit

**Comment by abpiskunov** [comment]
@dotnet/project-system  FYI

**Comment by abpiskunov** [comment]
:+1:

**Pull request merged**